### PR TITLE
Papa 7a deployments 

### PR DESCRIPTION
**Pull request by cdobs**
Deployed at Papa 7a:
GP05MOAS-GL363 D00004
GP05MOAS-PG566 D00002

FCPS: 3304-00160-00049 (363) and 3304-00170-00036 (566)
Deployment docs: 3304-00022-00061 (566) and 3304-00022-00062 (363)
Quick look report: 3203-00705

I also updated CUIDs for previous 363 deployments

### CODE REVIEW

File: deployment/GP05MOAS-GL363_Deploy.csv [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
+RB1605,,RB1605,,GP05MOAS-GL363-01-FLORDM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,50.16648,-144.50107,,1000,1000,
+RB1605,,RB1605,,GP05MOAS-GL363-02-DOSTAM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00161,50.16648,-144.50107,,1000,1000,
+RB1605,,RB1605,,GP05MOAS-GL363-04-CTDGVM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-09065,50.16648,-144.50107,,1000,1000,
+ZP20200930,,,,GP05MOAS-GL363-00-ENG000000,4,1,2020-10-02T12:59:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,46.01577,-133.00577,,1000,1000,
+ZP20200930,,,,GP05MOAS-GL363-01-FLORDM000,4,1,2020-10-02T12:59:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,46.01577,-133.00577,,1000,1000,
+ZP20200930,,,,GP05MOAS-GL363-02-DOSTAM000,4,1,2020-10-02T12:59:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,46.01577,-133.00577,,1000,1000,
+ZP20200930,,,,GP05MOAS-GL363-04-CTDGVM000,4,1,2020-10-02T12:59:00,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-00104,46.01577,-133.00577,,1000,1000,
```
Comment (pbrickley) [inline deployment/GP05MOAS-GL363_Deploy.csv:13]: good

File: deployment/GP05MOAS-PG566_Deploy.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+ZP20200930,,,,GP05MOAS-PG566-00-ENG000000,2,1,2020-10-02T12:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGCON-GLDRGP-00566,46.011,-133.00372,,200,200,
+ZP20200930,,,,GP05MOAS-PG566-01-CTDGVM000,2,1,2020-10-02T12:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-CTDGVM-09288,46.011,-133.00372,,200,200,
+ZP20200930,,,,GP05MOAS-PG566-02-DOSTAM000,2,1,2020-10-02T12:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-DOSTAM-00537,46.011,-133.00372,,200,200,
+ZP20200930,,,,GP05MOAS-PG566-03-FLORTM000,2,1,2020-10-02T12:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-FLORTM-04039,46.011,-133.00372,,200,200,
+ZP20200930,,,,GP05MOAS-PG566-04-FLORTO000,2,1,2020-10-02T12:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-FLORTO-01333,46.011,-133.00372,,200,200,
+ZP20200930,,,,GP05MOAS-PG566-05-NUTNRM000,2,1,2020-10-02T12:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-NUTNRM-00350,46.011,-133.00372,,200,200,
+ZP20200930,,,,GP05MOAS-PG566-06-PARADM000,2,1,2020-10-02T12:45:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-PARADM-50201,46.011,-133.00372,,200,200,
```
Comment (pbrickley) [inline deployment/GP05MOAS-PG566_Deploy.csv:8]: ok

**Review by pbrickley — APPROVED** [review]

File: deployment/GP05MOAS-GL363_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [inline deployment/GP05MOAS-GL363_Deploy.csv:13]: looks good

File: deployment/GP05MOAS-PG566_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [inline deployment/GP05MOAS-PG566_Deploy.csv:8]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: deployment/GP05MOAS-PG566_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/GP05MOAS-PG566_Deploy.csv:8]: Looks good.

File: deployment/GP05MOAS-GL363_Deploy.csv [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+RB1605,,RB1605,,GP05MOAS-GL363-02-DOSTAM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00161,50.16648,-144.50107,,1000,1000,
+RB1605,,RB1605,,GP05MOAS-GL363-04-CTDGVM000,2,1,2016-06-30T02:30:00,2016-07-05T07:08:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-09065,50.16648,-144.50107,,1000,1000,
+ZP20200930,,ZP20200930,,GP05MOAS-GL363-00-ENG000000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,46.01577,-133.00577,,1000,1000,
+ZP20200930,,ZP20200930,,GP05MOAS-GL363-01-FLORDM000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,46.01577,-133.00577,,1000,1000,
+ZP20200930,,ZP20200930,,GP05MOAS-GL363-02-DOSTAM000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,46.01577,-133.00577,,1000,1000,
+ZP20200930,,ZP20200930,,GP05MOAS-GL363-04-CTDGVM000,4,1,2020-10-02T12:59:00,2020-10-02T19:50:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-CTDGVM-00104,46.01577,-133.00577,,1000,1000,
```
Comment (snwhite) [inline deployment/GP05MOAS-GL363_Deploy.csv:13]: The recovery on 10/2/2020 at 19:50 is not indicated in the QLR.  Which one is right? Was it recovered or not?

**Review by snwhite — CHANGES_REQUESTED** [review]
What GL363 recovered as indicated in the Deploy CVS, or not as indicated in the QLR?

File: deployment/GP05MOAS-GL363_Deploy.csv (hunk 3 above)
Comment (cdobs) [reply deployment/GP05MOAS-GL363_Deploy.csv:13]: Updated QLR to include recovery time for 363.

Comment (snwhite) [reply deployment/GP05MOAS-GL363_Deploy.csv:13]: Looks good... sadly...

**Review by snwhite — APPROVED** [review]

**Pull request merged**